tools/clang/test/CodeGenSPIRV/semantic.depth.ps.hlsl

```diff
@@ -0,0 +1,11 @@
+// Run: %dxc -T ps_6_0 -E main
+
+// CHECK:                 OpEntryPoint Fragment %main "main" %gl_FragDepth {{%\d+}}
+
+// CHECK:                 OpDecorate %gl_FragDepth BuiltIn FragDepth
+
+// CHECK: %gl_FragDepth = OpVariable %_ptr_Input_float Input
```

Comment:
We talked about this offline. the FragDepth decoration should be attributed to an output variable, not an input variable.